### DISCUSSION TITLE
A special word that finish in '3d' always change to '3'

### DISCUSSION THREAD

**Discussion by tecnicoespaniol** [freeplane/freeplane#2009] [Help] [answered]
Hi! Its a recent install. It is happening to me when i write "03d" > "3". I tried "93d" > "93". "a3d" don't change. Then i tried "9999999993d" > "9999999993". Seems with numbers only. And only happens with 'd' character.
This happens when i write the first in a node. The second appears when i push enter for validate.

**Comment by dpolivaev** [comment, marked as answer]
Check https://docs.freeplane.org/user-documentation/Data_recognition_and_data_formats.html

**Comment by tecnicoespaniol** [comment]
Thank you for you answer dpolivaev.
Yes. With "If you start typing with a single quote, ', then the text format is changed to "Text" too" was enough in https://docs.freeplane.org/user-documentation/Data_recognition_and_data_formats.html#how-to-switch-data-recognition-off
